localstack/services/cloudformation/deployment_utils.py

```diff
@@ -4,10 +4,9 @@
 
 from localstack.config import dirs
 from localstack.utils import common
-
-# URL to "cfn-response" module which is required in some CF Lambdas
 from localstack.utils.common import select_attributes, short_uid
 
+# URL to "cfn-response" module which is required in some CF Lambdas
```

Comment:
add: the purpose of cfn-response is to make it easier to write "inline" code for custom resources.

Actually we should probably just copy the whole content here since it's really just a few lines of python/node code.